clients/ptranslator/README.adoc

```diff
@@ -58,7 +55,6 @@ to `gmond` udp mlticast packets.
 For this to work, the packets from Ganglia need to have a large enough TTL to be seen on the host where you run pTrans.
 
 Excerpt from gmond.conf:
-
```

Comment:
Generally there should be an empty line before such blocks (especially when they are denoted with `[source,...]` as otherwise the syntax coloring does not work.